VotingApplication/VotingApplication.Web/Scripts/Modules/Poll/VoteOn-Poll.js

```diff
@@ -0,0 +1,16 @@
+﻿(function () {
+    'use strict';
+
+    angular
+        .module('VoteOn-Poll', ['VoteOn-Common', 'VoteOn-Account', 'VoteOn-Vote'])
+            .config([
```

Comment:
Just a small formatting note; we typically indent config/controller/directive etc. the same amount as the module.
i.e. .module and .config have the same indentation.